src/frontend/src/views/LandingPageV2/components/Hero/components/HeroEmailForm/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import FormSubmitArrow from "src/common/images/form-submit-arrow.svg";
+import { EmailInput, HeroEmailForm, SubmitButton, SubmitIcon } from "./style";
+
+export default function EmailForm(): JSX.Element {
+  const [enteredEmail, setEnteredEmail] = useState("");
+
+  function submitEmail(e: React.FormEvent<HTMLFormElement>) {
+    e.preventDefault();
+
+    const emailRegex =
+      /^(([^<>()\]\\.,;:\s@"]+(\.[^<>()\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+    if (emailRegex.test(enteredEmail)) {
+      window.open(
+        `https://airtable.com/shrblHnTRd9dtu6c0?prefill_Email=${enteredEmail}`, // <-- replace url with new airtable form
+        "_blank"
+      );
+    } else {
+      alert("Please enter a valid email address.");
```

Comment:
is this supposed to be a native alert, or do we want to do a on-brand modal here?